distributed/deploy/cluster.py

```diff
@@ -82,6 +83,8 @@ async def _close(self):
         if self.status == Status.closed:
             return
 
+        self._adaptive.stop()
```

Comment:
Since `self._adaptive` is only set after `Cluster.adapt` is called, we might want to add a `with suppress(AttributeError):` here